addons/web/static/src/views/pivot/pivot_renderer.xml

```diff
@@ -77,14 +79,22 @@
                     t-on-add-custom-group-by="onAddCustomGroupBy(isXAxis ? 'col' : 'row')"
                 />
             </t>
+            <t t-if="l10n.direction === 'ltr'">
+                <i t-if="cell.isLeaf" class="oi oi-add--thick mr-1"/>
+                <i t-else="!cell.isLeaf" class="oi oi-subtract--thick mr-1"/>
+            </t>
             <span t-esc="cell.title"/>
+            <t t-if="l10n.direction === 'rtl'">
+                <i t-if="cell.isLeaf" class="oi oi-add--thick mr-1"/>
+                <i t-else="!cell.isLeaf" class="oi oi-subtract--thick mr-1"/>
+            </t>
         </th>
     </t>
 
     <t t-name="web.PivotMeasure" owl="1">
-        <th class="text-muted" t-att-colspan="cell.width" t-att-rowspan="cell.height" t-att-class="{
-                o_pivot_origin_row: cell.originIndexes,
-                o_pivot_measure_row: !cell.originIndexes,
+        <th class="text-muted text-center text-nowrap font-weight-normal bg-100" t-att-colspan="cell.width" t-att-rowspan="cell.height" t-att-class="{
+                'o_pivot_origin_row o_cursor_pointer': cell.originIndexes,
+                'o_pivot_measure_row o_cursor_pointer': !cell.originIndexes,
                 o_pivot_sort_order_asc: cell.order === 'asc',
                 o_pivot_sort_order_desc: cell.order === 'desc',
```

Comment:
There is not need of `o_cursor_pointer` class when   `o_pivot_sort_order_x` is applied ?

Reply:
No, because `o_pivot_sort_order_x` seems to be always added to `o_pivot_origin_ro or `o_pivot_measure_row` rather than replace them. 👍